README.md

```diff
@@ -1,103 +1,103 @@
-# 🎥 dolly
```

Comment:
I'm sure that you didn't want to change this entire file to CRLF!

Reply:
WHAT IN THE... ARGH. Thanks.

Reply:
Just Windows things™? :grimacing:

Reply:
Yup... Had autocrlf disabled due to some recent issues when running shell scripts from a Linux container inside a repo synced on Windows... CRLF is a gift that keeps on giving.

Reply:
Phew, fixed. Using the classic method of "screw everything, re-download the file from `main`, and apply fixes on top".

Reply:
`dos2unix`? Works every time :stuck_out_tongue:

Reply:
😲 <- mfw I realize I have it in Windows.

Reply:
MSYS2?

Reply:
git 👀

Reply:
git-for-windows is effectively MSYS2 with changes / preincluded packages / optimized `git` package specifically for Windows.

Reply:
![image](https://user-images.githubusercontent.com/16522064/200435430-c1b7091b-d576-4ff7-a7dc-98dc0116ec21.png)

Reply:
:joy: :joy: :joy: :joy: